dev-requirements.txt

```diff
@@ -16,6 +16,8 @@ readme-renderer~=24.0
 # https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-proto/pyproject.toml#L28
 grpcio-tools==1.48.1
 mypy-protobuf~=3.0.0
+opentelemetry-exporter-prometheus==1.12.0rc1
+protobuf~=3.18.1
```

Comment:
This shouldn't be `dev-requirements.txt`. It should be part of the example. Please create a simple `reqs.txt` in the Prometheus example directory.

Reply:
Fixed